docs/design/admission_control_limit_range.md

```diff
@@ -0,0 +1,122 @@
+# Admission control plugin: LimitRanger
+
+## Background
+
+This document proposes a system for enforcing min/max limits per resource as part of admission control.
```

Comment:
And it would have a sentence that explains _why_ one might want to setup a LimitRanger.